apps/web/components/ui/progress.tsx

```
@@ -0,0 +1,24 @@
+}
+
+export function Progress({ value = 0, className = '', indicatorClassName = '', ...props }: ProgressProps) {
+  return (
+    <div
+      role="progressbar"
+      aria-valuenow={value}
+      aria-valuemin={0}
+      aria-valuemax={100}
+      className={`relative w-full overflow-hidden rounded bg-muted ${className}`}
+      {...props}
+    >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "progressbar" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "progressbar".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)